MsweatherRequest/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+server.port=8080
```

Comment:
why to specify if this is the default port?

Reply:
done